packages/@react-aria/listbox/docs/useListBox.mdx

```diff
@@ -0,0 +1,422 @@
+* Support for mouse, touch, and keyboard interactions
+* Tab stop focus management
+* Keyboard navigation support including arrow keys, home/end, page up/down, select all, and clear
+* Automatic scrolling support during keyboard navigation
+* Typeahead to allow focusing options by typing text
+* Virtualized scrolling support for performance with long lists
+
+## Anatomy
+
+A listbox consists of a container element, with a list of options or groups inside.
+`useListBox`, `useOption`, and `useListBoxSection` handle exposing this to assistive
+technology using ARIA, along with handling keyboard, mouse, and interactions to support
+selection and focus behavior.
+
+`useListBox` returns props that you should spread onto the list container element:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useListBox.return.id].properties} />
+</TypeContext.Provider>
+
+`useOption` returns props for an individual option and its children:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useOption.return.id].properties} />
+</TypeContext.Provider>
+
+`useListBoxSection` returns props for a section:
+
+<TypeContext.Provider value={docs.links}>
+  <InterfaceType properties={docs.links[docs.exports.useListBoxSection.return.id].properties} />
+</TypeContext.Provider>
+
+State is managed by the <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} />
+hook from `@react-stately/list`. The state object should be passed as an option to 
+each of the above hooks.
+
+If a listbox, options, or group does not have a visible label, an `aria-label` or `aria-labelledby`
+prop must be passed instead to identify the element to assisitive technology.
+
+## State management
+
+`useListBox` requires knowledge of the options in the listbox in order to handle keyboard
+navigation and other interactions. It does this using 
+the <TypeLink links={collectionsDocs.links} type={collectionsDocs.exports.Collection} />
+interface, which is a generic interface to access sequential unique keyed data. You can 
+implement this interface yourself, e.g. by using a prop to pass a list of item objects, 
+but <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} /> from
+`@react-stately/list` implements a JSX based interface for building collections instead.
+
+In addition, <TypeLink links={statelyDocs.links} type={statelyDocs.exports.useListState} />
+manages the state necessary for multiple selection and exposes
+a <TypeLink links={selectionDocs.links} type={selectionDocs.exports.SelectionManager} />,
+which makes use of the collection to provide an interface to update the selection state.
+
+## Example
+
+This example uses HTML `<ul>` and `<li>` elements to represent the list, and applies
+props from <TypeLink links={docs.links} type={docs.exports.useListBox} />
+and <TypeLink links={docs.links} type={docs.exports.useOption} />.
+
+```tsx example
```

Comment:
this example isn't rendering for me https://reactspectrum.blob.core.windows.net/reactspectrum/3664b0363b110e7961d9058eaded72d813f0d48e/docs/react-aria/useListBox.html

Reply:
I'm getting some invalid hook calls

Reply:
Yep, it's due to the same parcel issue we had before when we disabled scope hoisting.